pages/1.13/security/ent/secrets/create-secrets/index.md

```diff
@@ -97,4 +97,38 @@ The contents of the file (referred to below as `my-secret.txt`) can be any text
   dcos security secrets create -f my-secret.txt developer/my-secret
   ```
 
-  <p class="message--important"><strong>IMPORTANT: </strong>The maximum file size for a secret is approximately one MB, subtracting approximately one KB for the secret store metadata.
+  <p class="message--important"><strong>IMPORTANT: </strong>The maximum file size for a secret is approximately one MB, subtracting approximately one KB for the secret store metadata.</p>
+
+# Creating secrets from a file via the DC/OS UI
```

Comment:
We should also update the screenshots for the `Creating key-value pair secrets using the UI` in this file. Specifically the modal `png` on line 51 is now out of date.

Reply:
Done.